keyboards/efreet/info.json

```
@@ -0,0 +1,15 @@
+{
+    "keyboard_name": "efreet",
+    "url": "",
+    "maintainer": "amberstarlight",
+    "width": 12,
+    "height": 4,
+    "layouts": {
+        "LAYOUT_2u": {
```

If you're going to enable the Community Layout support:

```suggestion
        "LAYOUT_planck_mit": {
```